### ISSUE TITLE
Problems installing bundle

### ISSUE THREAD

**Issue by Eragon666** [sonatra/SonatraSecurityBundle#6]
Hi, I am trying to setup our application with Symfony3.1 with the SonatraSecurityBundle. But after trying to do php app/console config:dump-reference SonatraSecurityBundle I get the following error:

The service "sonatra_security.expression.has_org_role.evaluator" has a dependency on a non-existent service "sonatra_security.organizational_context".

**Comment by francoispluchino** [comment]
You must enable the `organizational_context` option or disable the jms expression `has_org_role`:

```yaml
sonatra_security:
    expression:
        has_org_role: false
```

**Comment by francoispluchino** [comment]
I will disable by default the expression `has_org_role`.

**Comment by francoispluchino** [comment]
Fixed by a26c9750e68d11dbe7a1770d92ad0fe24b1fd567.

**Comment by Eragon666** [comment]
Thanks for the very quick answer and solution! The error is fixed, although I have another error now :( I will try to fix it, otherwise I will open another issue.
